### PR TITLE
10695 popup pensions content

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP10695](https://maps.tpondemand.com/entity/10695-replace-popup-chat-content-for-pensions)

This PR aims to replace the contents of the chat popup for pages which belong to the pensions category.  When a user opens the chat popup we will present them with specific webchat content.

### CODE REVIEW

File: app/views/shared/_webchat_popup.html.erb [hunk 1]
```diff
@@ -9,18 +9,20 @@
     </svg>
   </div>
   <div class="mobile-webchat__form">
-    <h4><%= t('contact_panels.chat_popup.title') %></h4>
+    <% if pensions_and_retirement_page? %>
+      <%= render 'shared/webchat_popup_pensions' %>
+    <% else %>
+    <h4><b><%= t('contact_panels.chat_popup.title') %><b></h4>
```
Comment (davidtrussler) [inline app/views/shared/_webchat_popup.html.erb]: Do we need the `<b>` tag here? Doesn't seem to make much difference if it's removed. However, if we do, the end tag is wrong (`<b>` not `</b>`).

File: app/views/shared/_webchat_popup_pensions.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<h4><b><%= t('contact_panels.chat_popup_pensions.title') %><b></h4>
+<h4><%= t('contact_panels.chat_popup_pensions.intro') %></h4>
```
Comment (davidtrussler) [inline app/views/shared/_webchat_popup_pensions.html.erb]: As above with the `<b>` tags (do we need them and the end tag is wrong). Not sure we should have 2 consecutive `<h4>`s either. Can we give these classes to deal with the styling if necessary?

File: app/views/shared/_webchat_popup_pensions.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<div class="mobile-webchat__button-container">
+  <div class="mobile-webchat__button">
+    <a class="button mobile-webchat__button--pensions disabled">
```
Comment (davidtrussler) [inline app/views/shared/_webchat_popup_pensions.html.erb]: I assume this is disabled until the work in a further ticket.

**Comment by davidtrussler** [comment]
@andreferreiradlw I've made a couple of minor comments here but otherwise I'm happy to say this is good to go. 👍

File: app/views/shared/_webchat_popup.html.erb (hunk 1 above)
Comment (andreferreiradlw) [reply app/views/shared/_webchat_popup.html.erb]: I've removed this tag now and added in some styles for the bold title text.

File: app/views/shared/_webchat_popup_pensions.html.erb (hunk 2 above)
Comment (andreferreiradlw) [reply app/views/shared/_webchat_popup_pensions.html.erb]: What should we have instead of the H4 tags?

**Pull request merged**